### PR TITLE
Enable Redis in kube-ops-view

### PR DESCRIPTION
**Pull request by tomislater**
#### What this PR does / why we need it:
Enable Redis in kube-ops-view

@hjacobs 

#### Checklist
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by davidkarlsen** [comment]
@tomislater can you fix the merge-conflict? TIA

**Comment by tomislater** [comment]
@davidkarlsen Done

File: stable/kube-ops-view/values.yaml [hunk 1]
```diff
@@ -39,11 +39,15 @@ rbac:
...
+# Defaults for Redis
 redis:
   enabled: false
   usePassword: false
   cluster:
     enabled: false
   master:
     persistence:
-      enabled: false
+      enabled: false
```
Comment (philoserf) [inline stable/kube-ops-view/values.yaml]: The end of file new line has been removed.

**Review by philoserf — CHANGES_REQUESTED** [review]

File: stable/kube-ops-view/values.yaml (hunk 1 above)
Comment (tomislater) [reply stable/kube-ops-view/values.yaml]:
```
stable/kube-ops-view/values.yaml
  54:1      error    too many blank lines (1 > 0)  (empty-lines)
```
lol

So, I am going to remove this line.

Comment (tomislater) [reply stable/kube-ops-view/values.yaml]: Ok, now there is only one line.

**Pull request merged**